Vowpal Wabbit - Contextual Bandits.ipynb

```diff
@@ -0,0 +1,705 @@
+      "\t|       conf       | number| search|dwnlded|evicted|| number|dwnlded|\n",
+      "\t---------------------------------------------------------------------\n",
+      "\t|      default     |   55  |   10  |   10  |   4   ||   51  |   0   |\n",
+      "\t---------------------------------------------------------------------\n",
+      "\n",
+      ":: problems summary ::\n",
+      ":::: ERRORS\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\tunknown resolver null\n",
+      "\n",
+      "\n",
+      ":: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS\n",
+      ":: retrieving :: org.apache.spark#spark-submit-parent-1317ef6b-13a4-4cff-af2b-208f5df7692f\n",
+      "\tconfs: [default]\n",
+      "\t0 artifacts copied, 51 already retrieved (0kB/13ms)\n",
+      "23/01/16 13:32:56 WARN SparkConf: The configuration key 'spark.yarn.user.classpath.first' has been deprecated as of Spark 1.3 and may be removed in the future. Please use spark.{driver,executor}.userClassPathFirst instead.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "23/01/16 13:32:56 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable\n",
+      "Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties\n",
+      "Setting default log level to \"WARN\".\n",
+      "To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).\n",
+      "23/01/16 13:32:56 WARN SparkConf: The configuration key 'spark.yarn.user.classpath.first' has been deprecated as of Spark 1.3 and may be removed in the future. Please use spark.{driver,executor}.userClassPathFirst instead.\n",
+      "23/01/16 13:32:56 WARN SparkConf: The configuration key 'spark.yarn.user.classpath.first' has been deprecated as of Spark 1.3 and may be removed in the future. Please use spark.{driver,executor}.userClassPathFirst instead.\n",
+      "23/01/16 13:32:56 WARN SparkConf: The configuration key 'spark.yarn.user.classpath.first' has been deprecated as of Spark 1.3 and may be removed in the future. Please use spark.{driver,executor}.userClassPathFirst instead.\n",
+      "23/01/16 13:32:57 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "# Bootstrap Spark Session\n",
+    "spark = (\n",
+    "    SparkSession.builder.config(\n",
```

Comment:
No need for this fancy spark session bootstrap, please use one from other notebooks

Reply:
removed file (was in wrong folder)